crates/turborepo-lib/src/run/summary/mod.rs

```diff
@@ -272,24 +272,24 @@ impl RunTracker {
 // not be used anywhere else.
 #[derive(Serialize)]
 #[serde(rename_all = "camelCase")]
-struct SinglePackageRunSummary<'a, 'b> {
+struct SinglePackageRunSummary<'a> {
```

Comment:
Since all of the fields in the `RunSummary` struct are [covariant](https://doc.rust-lang.org/beta/reference/subtyping.html#variance), `RunSummary` itself is covariant and we can simplify the struct to a single lifetime.

TL;DR
This is constructed from a `&'b RunSummary<'a>` and this implies that `'a` outlives `'b` (otherwise we would have a reference to a `RunSummary` that contains references that are no longer valid). In Rust you can always shorten a lifetimes so we shorten `RunSummary<'a>` to `RunSummary<'b>` and are left with a single lifetime.